File: Pogram/MapAudit.WepApi/MapAudit.DataAccess/Models/Carrier.cs
namespace MapAudit.DataAccess.Models
{
    public class Carrier : BaseEntity
    {
        public string Name { get; set; }
        public int DrivingExpirienceYears { get; set; }
        public TruckType TruckType { get; set; }
        public int FuelConsumptionInLitersInTownPer100Km { get; set; }
        public int AverageSpeedPerKmInTown { get; set; }
        public long? CompanyId { get; set; }
        public Company Company { get; set; }
        public List<Order> Orders { get; set; } 
    }
}

File: Pogram/GATravelsman/VisualiationApp/GAVisualization/src/app/models/carrier.ts
import { Order } from "./order";
import { TruckType } from "./truckType";


export class Carrier {
    id: number = 0;
    name: string ="";
    info: string = "";
    truckType!: TruckType;
    fuelConsumptionInLitersInTownPer100Km: number = 0;
    drivingExpirienceYears: number = 0;
    companyId: number = 0;
    orders: Order[] | undefined;
}